client/lib/abtest/active-tests.js

```diff
@@ -37,8 +37,8 @@ module.exports = {
 	signupStepOneCopyChanges: {
```

Comment:
Yeah, I'm just waiting for translations and then I'll factor it out once I have them. I'll keep it as is :)